wishlistPage/wishlistPage.html

```diff
@@ -0,0 +1,128 @@
+                        <h2 class="product-card-amount">XXXX$</h2>
+                        <button class="product-card-btn">
+                            Move to cart
+                        </button>
+                    </div>
+
+                    <div class="product-card">
+                        <img src="https://picsum.photos/300/375" alt="" srcset="" class="product-img">
+                        <span class="product-name">product name</span>
+                        <h2 class="product-card-amount">XXXX$</h2>
+                        <button class="product-card-btn">
+                            Move to cart
+                        </button>
+                    </div>
+
+                    <div class="product-card">
+                        <img src="https://picsum.photos/300/375" alt="" srcset="" class="product-img">
+                        <span class="product-name">product name</span>
+                        <h2 class="product-card-amount">XXXX$</h2>
+                        <button class="product-card-btn">
+                            Move to cart
+                        </button>
+                    </div>
+
+                    <div class="product-card">
+                        <img src="https://picsum.photos/300/375" alt="" srcset="" class="product-img">
+                        <span class="product-name">product name</span>
+                        <h2 class="product-card-amount">XXXX$</h2>
+                        <button class="product-card-btn">
+                            Move to cart
+                        </button>
+                    </div>
+
+                    <div class="product-card">
+                        <img src="https://picsum.photos/300/375" alt="" srcset="" class="product-img">
+                        <span class="product-name">product name</span>
+                        <h2 class="product-card-amount">XXXX$</h2>
+                        <button class="product-card-btn">
+                            Move to cart
+                        </button>
+                    </div>
+
+             </div>
+             
+        </div>
+    
+        <footer>
+            <footer class="footer">
+                <div class="footer-heading "><h3>You can contact me on below links</h3></div>
+               
+                <div class="footer-links">
+                    <a class="link" href="https://www.linkedin.com/in/tushar-bhandirge-300249118/">
+                        <img class="icon-image" src="./Image_resources/icons8-linkedin-30.png" />
+                    </a>
+               
+                    <a class="link" href="https://twitter.com/TusharBhandirge?s=09">
+                        <img class="icon-image" src="./Image_resources/icons8-twitter-30.png" />
+                    </a>
+               
+                    <a class="link" href="https://github.com/TusharBhandirge?tab=repositories">
+                        <img class="icon-image" src="./Image_resources/icons8-github-30.png"/>
```

Comment:
hii tushar check with your image source link for footer. other than that UI looks good👍

Reply:
okay i'll correct this image source. Thank!